pytorch_lightning/trainer/connectors/optimizer_connector.py

```diff
@@ -65,13 +65,20 @@ def update_learning_rates(self, interval: str, monitor_metrics=None):
                             RuntimeWarning,
                         )
                         continue
+
                 # update LR
-                old_lr = lr_scheduler['scheduler'].optimizer.param_groups[0]['lr']
                 if lr_scheduler['reduce_on_plateau']:
+                    old_lr = lr_scheduler['scheduler'].optimizer.param_groups[0]['lr']
```

Comment:
well, mind remind me what is the advantage of using `get_last_lr`?
atm, I am not sure if there rea a benefit of adding this complex solution to use the last PT api
cc: @SkafteNicki @awaelchli thought?

Reply:
`old_lr` and `new_lr` are only used in the `dev_debugger` right? Are they even nessesary?

Reply:
if it's doing the same thing internally, I don't see any point of updating it.
https://github.com/pytorch/pytorch/blob/c215ffb6a2de1c85a62a146a9e282908335ba3ed/torch/optim/lr_scheduler.py#L96-L99
https://github.com/pytorch/pytorch/blob/c215ffb6a2de1c85a62a146a9e282908335ba3ed/torch/optim/lr_scheduler.py#L166